Misc/NEWS.d/next/Library/2021-11-03-06-33-29.bpo-45689.nwTzgP.rst

```diff
@@ -0,0 +1 @@
+It allows us to give the custom Name for the threads submitted in ThreadPool
```

Comment:
Can you update this?  Please look at existing news entries to see what writing style and markup are used.